wp-content/themes/goonj-crm/functions.php

```diff
@@ -451,6 +455,7 @@ function goonj_redirect_after_individual_creation() {
 
 	$creationFlow = $individual['Individual_fields.Creation_Flow'];
 	$source = $individual['source'];
+	$sourceProcessingCenter = $individual['Individual_fields.Source_Processing_Center'];
```

Comment:
let's move this variable inside the cases where we needed it (`case 'office-visit'` and `case 'office-visit-contribution'`)